ssl/statem/extensions.c

```diff
@@ -948,7 +948,6 @@ static int final_server_name(SSL *s, unsigned int context, int sent)
      * was successful.
      */
     if (s->server) {
-        /* TODO(OpenSSL1.2) revisit !sent case */
```

Comment:
[side note: this comment  comes from 2c0267fdc99f8a06cb205f0faecc2ff06f0de8bf by me, with the implication that we want to make ```SSL_get_servername()``` return NULL on the client when the server has declined to accept SNI.  Now that we've documented the current "what the client has set takes precedence" behavior, there's no need for this different future direction, so removing the comment is correct.  New APIs would be the way to go down such a path, if still desired.]